src/pages/Workouts.module.css

```diff
@@ -0,0 +1,17 @@
+.workout {
+  background-image: url("../assests/images/workout.jpg");
+  background-size: cover;
+  min-height: 62vh;
```

Comment:
What is 62vh doing? This is a little suspect. It's saying "make the height 62% of the screen size", which seems kind of strange. Does this work at different resolutions?

Reply:
i took it out. I don't need it anymore